### ISSUE TITLE
When I just do_predict, I got KeyError of optimizer

### ISSUE THREAD

**Issue by allenyummy** [huminghao16/MTMSN#9]
When I just do_predict, I got KeyError of optimizer, like this:

![image](https://user-images.githubusercontent.com/36063123/68360085-931ba100-0159-11ea-9e23-82eb5c1d5f22.png)

Do you have any idea for the problem ?

**Comment by huminghao16** [comment]
The reason may be that the pre-trained model I uploaded does not contain the parameter for optimizer. So if you want to do prediction, you should mask out the following codes in the run file:
```
    global_step, global_epoch = 0, 1
    if os.path.isfile(save_path):
        checkpoint = torch.load(save_path)
        optimizer.load_state_dict(checkpoint['optimizer'])
        logger.info("Load optimizer from finetuned checkpoint: '{}' (step {}, epoch {})"
                    .format(save_path, checkpoint['step'], checkpoint['epoch']))
        global_step = checkpoint['step']
        global_epoch = checkpoint['epoch'] + 1
```
